portal-kernel/test/unit/com/liferay/portal/kernel/util/SystemPropertiesTest.java

```diff
@@ -58,6 +69,33 @@ public void testGetProperties() {
 		Assert.assertEquals(_TEST_VALUE, properties2.get(_TEST_KEY));
 	}
 
+	@Test
+	public void testGetPropertiesWithPrefix() {
+		Map<String, String> propertiesWithPrefix =
+			SystemProperties.getProperties(_PREFIX, false);
+
+		Assert.assertTrue(propertiesWithPrefix.isEmpty());
+
+		SystemProperties.set(_TEST_KEY, _TEST_VALUE);
```

Comment:
@W2jx It's still not obvious that `_TEST_KEY` is `_PREFIX` concating `_KEY`. I think it's better to write `_PREFIX + _KEY` as the key here.

Reply:
I will fix it.